playground/flow.php

```
@@ -18,40 +18,39 @@
   ),
-  'install' => Prompty::confirm('Install dependencies?', description: 'Runs npm install after scaffolding.'),
+  'send' => Prompty::confirm('Send order?', description: 'Passes the order to the kitchen.'),
 ],
-  intro: 'Create your project',
+  intro: 'Compose an order',
   outro: function (array $results): void {
-    Prompty::outro("You're all set!");
+    Prompty::outro('Order sent!');
     echo "\nCollected answers:\n";
     foreach ($results as $key => $value) {
       $display = is_array($value) ? (count($value) > 0 ? implode(', ', $value) : 'none') : (is_bool($value) ? ($value ? 'yes' : 'no') : $value);
```

PHPStan (2.2.7)

[error] Parameter `#2` $array of function implode expects array<string>, array<mixed, mixed> given.


(argument.type)